v2/apiserver/internal/core/kubernetes/substrate.go

```diff
@@ -57,6 +59,17 @@ type SubstrateConfig struct {
 	// this information whenever it needs to tell another component where to find
 	// the API server.
 	APIAddress string
+	// DefaultWorkerImage is the name/version of the OCI image that will be used
```

Comment:
Could this comment be a bit more explicit about the expected formatting of the image name and version?

Reply:
Sure. On it.

Reply:
Would this work?

```
// DefaultWorkerImage is the name of the OCI image that will be used for the
// Worker pod's container[0] if none is specified in a Project's
// configuration. The expected format is [REGISTRY/][ORG/]IMAGE_NAME[:TAG].
```

Reply:
Perfect!